projects/js-packages/publicize-components/src/components/resharing-panel/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { useSelect } from '@wordpress/data';
+import { useEffect, useState } from 'react';
+import { store as socialStore } from '../../social-store';
+import { ShareStatus } from '../post-publish-share-status/share-status';
+import { ShareStatusModalTrigger } from '../share-status';
+import styles from './styles.module.scss';
+
+export const ReSharingPanel = () => {
+	const featureFlags = useSelect( select => select( socialStore ).featureFlags(), [] );
+	const shareStatus = useSelect( select => select( socialStore ).getPostShareStatus(), [] );
+
+	const [ reShareTimestamp, setReShareTimestamp ] = useState( null );
+
+	useEffect( () => {
+		if ( shareStatus.polling ) {
+			// Update the timestamp whenever polling becomes true
+			setReShareTimestamp( Date.now() );
```

Comment:
Should we instead update the timestamp here in seconds so that we don't have to calculate things later? It's possible that we may use that reshare timestamp at some other place and forget to change milliseconds to seconds. We should sandbox our UIs from these conversions as possible, i.e. convert as soon as the data is created/received and convert as late as the data is displayed.